### ISSUE TITLE
Prośba do mebebe o wtyczkę Canalplus Telewizja przez Internet

### ISSUE THREAD

**Issue by irkfine** [mbebe/blomqvist#420]
Jako posiadacz wysokiego pakietu Canal mogę też włączyć się do Telewizji przez Internet z ich strony. Mają tam zestaw kanałów na żywo podobny do NC+ Go ale jest też takie coś że można kliknąć na 4 kanały i oglądać je jednocześnie. Być może Telewizja przez Internet przesyła strumienie full 1080p a nie 720p tak jak NC Go? Na pewno byłaby pożyteczna wtyczka do tej Telewizji bo ten serwis będzie się rozszerzał więc prośba do mbebe o zrobienie tego. Oczywiście najlepiej od razu z generatorem listy:).

**Comment by hevet** [comment]
https://github.com/mbebe/blomqvist/tree/master/zips/plugin.video.canalplusvod

**Comment by irkfine** [comment]
[Canal+VOD](https://ncplusgo.pl/Collection/CatchUp/Details?CollectionCodename=catchup-canal-vod)
[Canal+na żywo](https://www.canalplus.com/pl/liveTV/)

To samo?

**Comment by hevet** [comment]
We wtyczce po zalogowaniu można oglądać kanały na żywo.

**Comment by kopernik1983** [comment]
> 
> 
> [Canal+VOD](https://ncplusgo.pl/Collection/CatchUp/Details?CollectionCodename=catchup-canal-vod)
> [Canal+na żywo](https://www.canalplus.com/pl/liveTV/)
> 
> To samo?

Nie nie to samo " Twój link Canal + VOD to NC+Go ( I do tego zresztą też masz wtyczkę w repo ) , wtyczka do " Canalplus Telewizja przez Internet " tez jest w repo i nazywa się Canal+ VOD i jest dostępna już od pół roku :)

**Comment by irkfine** [comment]
> Nie nie to samo " Twój link Canal + VOD to NC+Go ( I do tego zresztą też masz wtyczkę w repo ) , wtyczka do " Canalplus Telewizja przez Internet " tez jest w repo i nazywa się Canal+ VOD i jest dostępna już od pół roku :)

Tak, teraz się wyjaśniło z tymi nazwami VOD. Canal+ VOD to Telewizja przez Internet. Pytanie dodatkowe czy strumienie tej telewizji i NC+ Go dla tych samych kanałów są identyczne? 

I jak zwykle w tym wątku, generator listy byłby bardzo przydatny.

**Comment by kopernik1983** [comment]
Na pewno nie są identyczne :) dwa różne serwisy , choć zawartość będzie się bogacić i o to co jest w NC+ to stream na pewno nie będzie taki sam. 

> I jak zwykle w tym wątku, generator listy byłby bardzo  przydatny. 
> 

Nie widzę powodu , dla którego nie mógłbyś sobie takiej listy zrobić  :P dodajesz kanał do ulubionych , odpalasz , wchodzisz w log kodi i link gotowy :P Ale zapewne ci się nie chce :P więc możesz skorzystać z tego https://github.com/BlackGlassSkin/Polish-Vip-Tv :) :)

**Comment by irkfine** [comment]
> Ale zapewne ci się nie chce :P więc możesz skorzystać z tego https://github.com/BlackGlassSkin/Polish-Vip-Tv :) :)

Te informacje to dla mnie rewelacja, dzięki za link!  Podejrzenie że mi się nie chce jest bez pokrycia, ja po prostu pojęcia nie miałem o tych możliwościach bo gdybym wiedział to nie traciłbym czasu na wypisywanie próśb tutaj. Tylko gdzie zdobywa się te informacje bo na to się nie natknąłem tylko na generator listy do nc+ który od razu uruchomiłem? Teraz też zajrzałem do kodi.log (przypadkowo odkryłem gdzie on jest) i rzeczywiście z niego można jakoś wyciągnąć link do kanału ale dla 100 kanałów byłaby to duża robota bez generatora. Sprawdziłem Horizon Go i działa. Dzięki za link do gotowych list, do komlpletu przydałyby się Polsat Go, Orange Go i Vecra :).
